x-pack/plugins/siem/public/management/pages/policy/view/policy_list.test.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import * as reactTestingLibrary from '@testing-library/react';
+
+import { PolicyList } from './index';
+import { mockPolicyResultList } from '../store/policy_list/mock_policy_result_list';
+import { AppContextTestRender, createAppRootMockRenderer } from '../../../../common/mock/endpoint';
+import { AppAction } from '../../../../common/store/actions';
+
+describe('when on the policies page', () => {
+  let render: () => ReturnType<AppContextTestRender['render']>;
+  let history: AppContextTestRender['history'];
+  let store: AppContextTestRender['store'];
+
+  beforeEach(() => {
+    const mockedContext = createAppRootMockRenderer();
+    ({ history, store } = mockedContext);
+    render = () => mockedContext.render(<PolicyList />);
+  });
+
+  it('should show a table', async () => {
+    const renderResult = render();
+    const table = await renderResult.findByTestId('policyTable');
+    expect(table).not.toBeNull();
+  });
+
+  describe('when list data loads', () => {
+    let firstPolicyID: string;
+    beforeEach(() => {
+      reactTestingLibrary.act(() => {
+        history.push('/management/policy');
+        reactTestingLibrary.act(() => {
+          const policyListData = mockPolicyResultList({ total: 3 });
+          firstPolicyID = policyListData.items[0].id;
+          const action: AppAction = {
+            type: 'serverReturnedPolicyListData',
+            payload: {
+              policyItems: policyListData.items,
+              total: policyListData.total,
+              pageSize: policyListData.perPage,
+              pageIndex: policyListData.page,
+            },
+          };
+          store.dispatch(action);
+        });
+      });
+    });
+    it('should display rows in the table', async () => {
+      const renderResult = render();
+      const rows = await renderResult.findAllByRole('row');
+      expect(rows).toHaveLength(4);
+    });
+    it('should display policy name value as a link', async () => {
+      const renderResult = render();
+      const policyNameLink = (await renderResult.findAllByTestId('policyNameLink'))[0];
+      expect(policyNameLink).not.toBeNull();
+      expect(policyNameLink.getAttribute('href')).toContain(`policy/${firstPolicyID}`);
+    });
+  });
+});
```

Comment:
@kevinlog - the 3rd AC for policy status linking to policy response is not tested here in Endpoint List and Endpoint Details: https://github.com/elastic/endpoint-app-team/issues/402. Is that functionality part of this PR? I did not see it. If so, can we add a test here for it? Otherwise, I can add it to my regression test run as manual for now and create an automated test ticket.

Reply:
@aisantos this just makes changes to the policy list.  The changes to the endpoint list will come later, new links, info, etc